<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hrQtPro</class>
 <widget class="QWidget" name="hrQtPro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hrQtPro</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="9,1">
     <item>
      <widget class="QLabel" name="previewLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="faceNumLab">
           <property name="text">
            <string>人数：0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit">
           <property name="text">
            <string>www</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="IDlineEdit">
           <property name="text">
            <string>111</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="registerBtn">
           <property name="text">
            <string>拍照注册</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="picregisterBtn">
           <property name="text">
            <string>自动注册</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="delFaceBtn">
         <property name="text">
          <string>删除人脸</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
